projects/packages/jetpack-mu-wpcom/src/features/admin-color-schemes/admin-color-schemes.php

```diff
@@ -144,8 +146,12 @@ function reenqueue_core_color_scheme() {
 	$core_color_schemes = array( 'blue', 'coffee', 'ectoplasm', 'fresh', 'light', 'midnight', 'modern', 'ocean', 'sunrise' );
 	$color_scheme       = get_user_option( 'admin_color' );
 	if ( in_array( $color_scheme, $core_color_schemes, true ) ) {
-		// phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
-		wp_enqueue_style( 'jetpack-core-color-scheme', "https://s0.wp.com/wp-admin/css/colors/$color_scheme/colors.min.css" );
+		wp_enqueue_style(
```

Comment:
@fushar I had to do this because otherwise our overrides for the Core color schemes won't be loaded.

For instance, this how the sidebar notice would look with/without these changes for the Modern color scheme:

Before | After
--- | ---
<img width="179" alt="Screenshot 2024-04-22 at 17 39 11" src="https://github.com/Automattic/jetpack/assets/1233880/eb745b8b-81b0-4000-9f2b-c3377647a8c3"> | <img width="178" alt="Screenshot 2024-04-22 at 17 40 44" src="https://github.com/Automattic/jetpack/assets/1233880/b7b00f9e-4e1c-41ad-87bd-47a4fdc1d2f4">

Would this cause any issue?

Reply:
> Would this cause any issue?

Yup, unfortunately it will resurface the issue fixed by https://github.com/Automattic/jetpack/pull/36341. The overrides have different top admin bar  colors compared to Core's. (This is because the colors were being used for our own dotcom admin bar.) For example, with the Sunrise color scheme:

|Actual|Expected|
|-|-|
|<img width="396" alt="Screenshot 2024-04-23 at 13 17 42" src="https://github.com/Automattic/jetpack/assets/1525580/596f6c36-5726-40fb-9032-022eadee97e8">|<img width="391" alt="image" src="https://github.com/Automattic/jetpack/assets/1525580/867b9b18-69e0-4342-81ec-08a216468c7b">|

Notice that the text and icon in the top admin bar have slightly different color, and that it has an unexpected bottom border.

I'm thinking on how to fix this...

Reply:
I guess one way would be to split the overrides into two different files (masterbar.css and sidebar.css) and load only the ones that won't cause conflicts.

Reply:
Or we could also wrap the top bar styles in a selector that won't be matched on untangled sites. Example:

```
.is-default-admin-interface #wpadminbar
```

And the `is-default-admin-interface` is only added on sites with the default admin interface enabled.

Reply:
Yep, I was thinking on your first option but the second one looks interesting too. Let me know what you end up doing!

Reply:
@fushar ended up separating files so we can load the ones we need. See https://github.com/Automattic/jetpack/pull/37022